src/main/java/de/tum/cit/aet/service/artemis/interaction/SimulatedArtemisStudent.java

```diff
@@ -578,6 +585,46 @@ private List<RequestStat> solveAndSubmitProgrammingExercise(ProgrammingExercise
         return requestStats;
     }
 
+
+private List<RequestStat> solveAndSubmitFileUploadExercise(FileUploadExercise fileUploadExercise) {
+    List<RequestStat> requestStats = new ArrayList<>();
+    long start = System.nanoTime();
+    var participation = fileUploadExercise
+        .getStudentParticipations()
+        .iterator()
+        .next();
+    webClient
+        .get()
+        .uri(uriBuilder ->
+            uriBuilder.pathSegment("api", "fileupload", "participations", participation.getId().toString(), "file-upload-editor").build()
+        )
+        .retrieve()
+        .toBodilessEntity()
+        .block();
+    requestStats.add(new RequestStat(now(), System.nanoTime() - start, MISC));
+
+    ByteArrayResource file = FileGeneratorUtil.getDummyFile(1024 * 1024, "test-file.txt");
```

Comment:
Is there a deeper meaning behind the file size? Could we extract it to a variable and write a comment there whether the current size is quite close to a real submitted file / thoughts behind that file size?